ansible/roles/ocp-workload-aiedge-computing-blueprint/tasks/workload.yml

```diff
@@ -569,6 +571,18 @@
   command:
     cmd: "oc apply -f '{{ hub_repo_dir }}/base/03_services/argocd-operator-subs/subscription-labels.yaml'"
 
+- name: Create the ACM namespace for the blueprint-management-hub
+  command:
+    cmd: "oc apply -f '{{ hub_repo_dir }}/base/03_services/manuela-factory-subs/namespace.yaml'"
+
+- name: Create the ACM channel for the blueprint-management-hub repo
+  command:
+    cmd: "oc apply -f '{{ hub_repo_dir }}/base/03_services/manuela-factory-subs/channel.yaml'"
+
+- name: Create the ACM factory application placementrule
+  command:
+    cmd: "oc apply -f '{{ hub_repo_dir }}/base/03_services/manuela-factory-subs/placementrule-labels.yaml'"
+
 - name: Create the ACM subscription for MANUela factory
   command:
     cmd: "oc apply -f '{{ hub_repo_dir }}/base/03_services/manuela-factory-subs/subscription-labels-devcluster-factory.yaml'"
```

Comment:
`k8s`? There may be more locations...

Reply:
There are more locations. See https://github.com/redhat-cop/agnosticd/pull/3768#discussion_r633858238 about `lookup` not working on external files

Reply:
Updated to use the `k8s` module with `src` parameter